src/chain/polkadot/types/staking/events.ts

```diff
@@ -46,3 +128,25 @@ export const rewarded =  {
         })
     ),
 }
+
+export const slashed =  {
+    name: 'Staking.Slashed',
+    /**
+     *  One validator (and its nominators) has been slashed by the given amount.
+     *  \[validator, amount\]
```

Comment:
Is the "\[validator, amount\]" line intentional?

Reply:
Same here, everything in types folder is auto generated